packages/scm-extra/src/browser/history/scm-history-widget.tsx

```diff
@@ -166,21 +174,25 @@ export class ScmHistoryWidget extends ScmNavigableListWidget<ScmHistoryListNode>
         this.addListNavigationKeyListeners(this.node);
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         this.addEventListener<any>(this.node, 'ps-scroll-y', (e: Event & { target: { scrollTop: number } }) => {
-            if (this.listView && this.listView.list && this.listView.list.Grid) {
+            if (this.listView?.list) {
                 const { scrollTop } = e.target;
-                this.listView.list.Grid.handleScrollEvent({ scrollTop });
+                this.listView.list.scrollTo({
+                    top: scrollTop
+                });
             }
         });
     }
 
     override update(): void {
         if (this.listView && this.listView.list) {
-            this.listView.list.forceUpdateGrid();
+            this.listView.forceUpdate();
```

Comment:
If this option exists for the `listView`, should we leave `forceUpdate` in `TreeWidget` and delegate to this method? Or are their functions not really analogous, so that it would be more deceptive than helpful?

Reply:
I've removed the `update` override, since it didn't seem to have any influence on the behavior. The old `forceUpdate` was needed, since the `rows` weren't part of the properties, but they are now, so the component gets correctly notified once something changes.